test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
As far as I can tell, so far, the app-level setting had no effect whatsoever, and only the model level was actually sort of working, but only one level deep: nested routes were reverting to the default of not normalizing the paths, ignoring the model-level setting (as well as the app-level oe).